activiti-core/activiti-api-impl/activiti-api-process-runtime-impl/src/main/java/org/activiti/spring/process/ProcessVariablesInitiator.java

```diff
@@ -88,13 +88,12 @@ public ProcessVariablesInitiator(ProcessExtensionService processExtensionService
 
         if (processExtensionService.hasExtensionsFor(processDefinition)) {
 
-            processVariables = variablesCalculator.calculateOutPutVariables(MappingExecutionContext.buildMappingExecutionContext(
+            processVariables.putAll(variablesCalculator.calculateOutPutVariables(MappingExecutionContext.buildMappingExecutionContext(
```

Comment:
Is there any reason why passed vars at creation of the process and vars from extensions are mutually exclusive?